app/helpers/advice_page_helper.rb

```diff
@@ -0,0 +1,9 @@
+module AdvicePageHelper
+  def advice_page_path(school, advice_page, tab = :insights)
+    polymorphic_path([tab, school, :advice, advice_page.key.to_sym])
+  end
+
+  def advice_baseload_high?(val)
```

Comment:
This feels like it should not be a helper but be part of the benchmark service. I have no concept of how tricky this would be though so please ignore me as required :-)

Reply:
Yes good point - will look into whether this can be part of the service